src/CMakeLists.txt

```diff
@@ -325,7 +344,7 @@ if(BUILD_FAST_TOKENIZERS)
 else()
   set(fast_tokenezers_libs_dir "${fast_tokenizer_SOURCE_DIR}/third_party/lib")
   if(WIN32 AND X86_64)
-    set(extra_libs "${fast_tokenizer_SOURCE_DIR}/lib/core_tokenizers.dll")
+    set(extra_libs "${fast_tokenizer_SOURCE_DIR}/lib/core_tokenizers${CMAKE_DEBUG_POSTFIX}.dll")
```

Comment:
as far as I can see debug libraries are not provided in prebuilt package.
Hence, these changes are not required

Reply:
removed